spec/gilded_rose_spec.rb

```diff
@@ -29,12 +29,17 @@
       expect(items[0].quality).to eq (8)
     end
 
-    # NEED: to deal with negative quality input in item constructor
     it "quality should never be negative" do
-      items = [Item.new("foo", 0, 0), Item.new("foo", 0, -1)]
+      items = [Item.new("foo", 0, 0)]
+      GildedRose.new(items).update_quality()
+      GildedRose.new(items).update_quality()
+      expect(items[0].quality).to eq (0)
+    end
+
+    xit "if item is provided a negative quality value, it will return a quality of 0" do
+      items = [Item.new("foo", 0, -1)]
```

Comment:
If `foo` is replaced with an item name (for example, `aged brie`, `backstage passes`), will the behaviour be the same?

Reply:
I see where you are going with this - thanks you!